openslides/assignments/static/js/assignments/site.js

```diff
@@ -40,6 +40,9 @@ angular.module('OpenSlidesApp.assignments.site', [
                     assignments: function(Assignment) {
                         return Assignment.findAll();
                     },
+                    users: function(User) {     //TODO: Load only related users cause of better performance
+                        return User.findAll();
+                    },
```

Comment:
This is _a_ solution for [this isse](https://github.com/OpenSlides/OpenSlides/issues/2581).
What would _probably_ be a better solution would be something like

```
angular.forEach($scope.assignmentsFiltered, function(assignment) {
    Assignment.loadRelations(assignment, 'assignment_related_users');
    ...
});
```
after the click on the PDF-Button. That should only re-load users needed for the elections. Unfortionatly, this causes an error.